Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Backtrace",
+    platforms: [
+        .iOS(.v10),
```

Comment:
Where did you get these minimum versions from? They look good, but just checking

Reply:
We have taken the refrence of exisitng targets like Example-iOS, Backtrace-iOS etc.

Reply:
Can you please double check that there version are indeed supported by the latest version of PLCrashreporter?

Reply:
Its been verified again.

Following version supported by PLCrashreporter latest.
platforms: [
        .iOS(.v9),
        .macOS(.v10_10),
        .tvOS(.v9)
    ],